packages/ui/src/components/va-input/hooks/useCleave.ts

```diff
@@ -78,7 +78,7 @@ export const useCleave = (
       }
     }
 
-    return syncValue.value
+    return syncValue.value || ''
```

Comment:
Expect number (0) to work correctly here.

```suggestion
    return syncValue.value ?? ''
```